sdk/identity/azure-identity/azure/identity/_credentials/azure_powershell.py

```diff
@@ -136,6 +136,12 @@ def get_token(
           receive an access token
         """
 
+        # Check if claims challenge is provided
+        if claims:
+            raise CredentialUnavailableError(
+                message=f"Fail to get token with claims challenge '{claims}', please run Connect-AzAccount --ClaimsChallenge"
```

Comment:
What we want is 

message=f"Fail to get token, please run Connect-AzAccount --ClaimsChallenge {claims}"